demisto_sdk/dev_tools/runner.py

```diff
@@ -0,0 +1,156 @@
+
+        return ans
+
+    def run(self):
+        """Run the integration command on the remote Demisto instance
+        and pretty prints the result.
+        """
+        playground_id = self._get_playground_id()
+
+        query = self.query
+        if self.debug_mode:
+            query = self.query + ' debug-mode="true"'
+
+        log_ids = self._run_query(playground_id, query)
+
+        if self.debug_mode:
+            if not log_ids:
+                print_warning('Entry with debug log not found')
+            else:
+                self._export_debug_log(log_ids)
+
+    def _get_playground_id(self):
+        """Retrieve Playground ID from the remote Demisto instance.
+        """
+        filter = {'filter': {'type': [9]}}
+        ans = self.client.search_investigations(filter=filter)
+
+        if ans.total != 1:
+            print_error(
+                'Got unexpected amount of results. If you are using MT environment, '
+                'use the Tenant as Demisto\'s URL instead of the Master.'
+            )
+            raise RuntimeError(
+                f'Got unexpected amount of results in getPlaygroundInvestigationID. '
+                f'Response was: {ans.total}'
+            )
+
+        if ans.data is None:
+            raise RuntimeError(f'Missing data field in response')
+
+        result = ans.data[0].id
+
+        print_v(f'Playground ID: {result}', self.log_verbose)
+
+        return result
+
+    def _run_query(self, playground_id: str, query: str):
+        """Run a query on the Playground of the remote Demisto instance.
+        Print the readable output and return the id of the debug log file.
+        """
+        update_entry = {
+            'investigationId': playground_id,
+            'data': query
+        }
+        ans = self.client.investigation_add_entries_sync(update_entry=update_entry)
+
+        log_ids = []
+
+        for entry in ans:
+            # ans should have entries with `contents` - the readable output of the command
+            if entry.parent_content:
+                print_color('### Command:', LOG_COLORS.YELLOW)
+                print(entry.parent_content)
+            if entry.contents:
+                print_color('## Readable Output', LOG_COLORS.YELLOW)
+                if entry.type == 4:  # entryType is 'error'
+                    print_error(entry.contents + '\n')
+                else:
+                    print(entry.contents + '\n')
+            else:
+                print_warning('No readable output for query\n')
+
+            # and entries with `file_id`s defined, that is the fileID of the debug log file
+            if self.debug_mode and entry.file_id:
+                log_ids.append(entry.id)
+
+        return log_ids
+
+    def _export_debug_log(self, log_ids: list):
+        """Retrieve & rexport debug mode log files
+
+        Args:
+            log_ids (list): artifact ids of the log files
+        """
+        if self.debug_mode == '-':
```

Comment:
in which case will this be `-` it should be a bool

Reply:
not a bool anymore